br/pitr-monitoring-and-alert.md

```diff
@@ -0,0 +1,107 @@
+| **tikv_log_backup_initial_scan_operations**           | Counter   | Statistics of RocksDB-related operations during initial scan. <br/>`cf :: {"default", "write", "lock"}, op :: RocksDBOP`                                                                       |
+| **tikv_log_backup_enabled**                           | Counter   |  Whether to enable log backup. If the value is greater than `0`, log backup is enabled.                                                                                                                                |
+| **tikv_log_backup_observed_region**                   | Gauge     | The number of Regions being listened to.                                                                                                                                        |
+| **tikv_log_backup_task_status**                       | Gauge     | The status of the log backup task. `0` means running. `1` means paused. `2` means error.  <br/>`task :: string`                                                                                                |
+| **tikv_log_backup_pending_initial_scan**              | Gauge     | Statistics of pending initial scans. <br/>`stage :: {"queuing", "executing"}`                                                                                                     |
+
+## Grafana configuration
+
+- For clusters deployed using TiUP, the [Grafana](https://grafana.com/) dashboard contains the PITR panel. The **Backup Log** panel in the TiKV-Details dashboard is the PITR panel.
+- For clusters deployed manually, refer to [Import a Grafana dashboard](/deploy-monitoring-services.md#step-2-import-a-grafana-dashboard) and upload the [tikv_details](https://github.com/tikv/tikv/blob/master/metrics/grafana/tikv_details.json) JSON file to Grafana. Then find the **Backup Log** panel in the TiKV-Details dashboard.
+
+## Alert configuration
+
+Currently, PITR does not have built-in alert items. This section introduces how to configure alert items in PITR and recommends some items.
+
+To configure alert items in PITR, follow these steps:
+
+1. Create a configuration file (for example, `pitr.rules.yml`) for the alert rules on the node where Prometheus is located. In the file, fill in the alert rules according to the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/), the following recommended alert items, and configuration sample.
+2. In the `rule_files` field of the Prometheus configuration file, add the path of the alert rule file.
+3. Send `SIGHUP` signal to the Prometheus process (`kill -HUP pid`) or send HTTP POST request to `http://prometheus-addr/-/reload` (before you send the HTTP request, add the `--web.enable-lifecycle` parameter when starting Prometheus).
```

Comment:
```suggestion
3. Send `SIGHUP` signal to the Prometheus process (`kill -HUP pid`) or send an HTTP `POST` request to `http://prometheus-addr/-/reload` (before you send the HTTP request, add the `--web.enable-lifecycle` parameter when starting Prometheus).
```
[Some specific items to put in code font](https://developers.google.com/style/code-in-text?hl=en#:~:text=HTTP%20verbs,content%2Dtype%20values)